docs/app-directory/overview.md

```diff
@@ -7,18 +7,75 @@ hide_title: true
 
 # App Directory Overview
 
-The FDC3 App Directory provides trusted identity for financial desktop apps. This identity can be used to prevent spoofing and man-in-the-middle attacks when apps communicate with one another and exchange data. The App Directory also enables service discovery. Apps are registered with a declaration of the intents and context data that can be used when interoperating.
+An Application Directory (AppD) is a structured repository of information about apps that can be used in an FDC3-enabled desktop. In other words, it’s a convenient way of storing and managing metadata about apps in your ecosystem. 
 
-## Core features
+The application metadata stored in appD includes the app name, type, details about how to run the application, its icons, publisher, support contact details and so on. All this information is readily available in one place and can be used both to populate a launcher or app catalog UI for your users , and by the desktop agent managing the apps on your desktop. In fact, if your smart desktop follows the FDC3 standard, appD is the primary way your desktop agent knows all the details about apps available to run on your desktop. Conversely, if an app is not listed in appD, the desktop agent can’t ensure its participation in context sharing or use it to resolve intents.
+
+
+## Advantages 
+
+Using appD offers many advantages both for the financial institutions running a smart desktop and for vendors that provide FDC3-compliant apps:
+
+
+### For the user
+
+
+#### Easier to find app info
+
+AppD is the one place to collect all the information about apps. The more apps you have, the more you’ll appreciate the convenience of not having to chase down details about each. This is particularly important for large institutions with multiple desks.
+
+
+#### Human readable
+
+AppD has two kinds of users. One is the desktop agent, but the other is humans administrating and using the smart desktop at your organization. Hence, AppD contains information about apps in both machine- and  human-readable forms. For example, it includes both a unique identifier for the app that is used to refer to it in code and a human-friendly app name, icon, description and tooltip necessary to populate a launcher menu or app catalog user interface. 
+
+
+#### Apps are discoverable
+
+For large institutions, it can be difficult to keep track of all the apps (developed both in-house and by vendors), since a typical desktop could have many. Users can search appD to discover the apps they need. It may already reside on their system or be available to them over the internet, but if they don’t have a way to search the apps available to them, they won’t be able to find it. AppD provides a way for users to discover the apps they need.
```

Comment:
```suggestion
For large institutions, it can be difficult to keep track of all the apps (developed both in-house and by vendors), since a typical desktop could have many. Users can search appD to discover the apps they need. An app might already reside on their system or be available to them over the internet, but if they don’t have a way to search the apps available to them, they won’t be able to find it. An appD provides a way for users to discover the apps they need.
```